packages/opentelemetry-exporter/package.json

```diff
@@ -53,11 +53,11 @@
   ],
   "dependencies": {
     "@instana/serverless": "1.137.1",
+    "@opentelemetry/api": "1.0.3",
```

Comment:
@willianpc After implementing the new eslint rule, it complained about this line: https://github.com/instana/nodejs/blob/1d3b29806f1e423f12a4b5cb7340ef584f2991f5/packages/opentelemetry-exporter/src/index.js#L10

There we import `@opentelemetry/api` in production code, but so far we only had it in our `devDependencies`. This actually works anyway, because we depend on `@opentelemetry/core` already and that one depends on  `@opentelemetry/api` in turn. So we always have the `api` package available implicitly, even when `@instana/opentelemetry-exporter` is installed as a dependency or `npm install --production` is used etc.

I know we discussed the OTel dependencies to some extent here already: https://github.com/instana/nodejs/pull/388/files#r712224844 - but not specifically the `api` module (or I cannot remember).

Thoughts? Should we rather leave `@opentelemetry/api` in the devDependencies and just disable the new rule for that line in index.js? That would also work for me. Or do we want to move it from devDependencies to dependencies?

Reply:
This is wrong then. Could you move `@opentelemetry/api` to dependencies?
Essentially, this is the otel exporter own logger, so it makes sense to be in production, as our `instanaBackendConnector` will forward any message to this one.
I don't know how this went unnoticed. 
Thanks for the finding!

Reply:
> Could you move @opentelemetry/api to dependencies?

I actually already have (see the diff), so I guess this change is okay then.